daemon/src/projection.rs

```diff
@@ -954,6 +944,46 @@ pub enum TxLabel {
     Collaborative,
 }
 
+struct AnnualisedFundingRate(Decimal);
+
+impl From<FundingRate> for AnnualisedFundingRate {
+    fn from(funding_rate: FundingRate) -> Self {
+        Self(
+            funding_rate
+                .to_decimal()
+                .checked_mul(Decimal::from(
+                    (24 / SETTLEMENT_INTERVAL.whole_hours()) * 365,
+                ))
+                .expect("not to overflow"),
+        )
+    }
+}
+
+impl fmt::Display for AnnualisedFundingRate {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.0.round_dp(2).fmt(f)
+    }
+}
```

Comment:
There is a module somewhere that makes it very easy to format a `Decimal` with 2 decimal places in a `serde` serializer.

If we would implement `Serialize` for this instead, you could just store `AnnualisedFundingRate` within `CfdOrder` and stay in type-safe land :)

Reply:
yes, I've seen that! I'll make a follow-up with an Option of the strong type on top of @da-kami 's changes. Added to my list for the day :)